### PR TITLE
configured react modal and added login modal

### LINKED ISSUE #14: React modal template

### PR DESCRIPTION
**Pull request by eric-ls**
Fix #14 

screenshot 2016-09-26 11 40 56

### CODE REVIEW

**Comment by chenhoward** [comment]
@eric-ls is da whole team

File: app/assets/javascripts/components.js [hunk 1]
```diff
@@ -1 +1,2 @@
+//= require_tree ./templates
```
Comment (warrenshen) [inline app/assets/javascripts/components.js]: Ayy.

File: app/assets/javascripts/components/login/home_login.jsx [hunk 2]
```diff
@@ -1,19 +1,62 @@
...
+    console.log("Handling login...");
   }
 
   render() {
     return (
-      <span>
-        <button className="button button--white splash-login-button">Log In</button>
-      </span>
+      <div>
+        <button className={`button ${this.props.btnClasses} splash-login-button`}
```
Comment (warrenshen) [inline app/assets/javascripts/components/login/home_login.jsx]: Nit: it's a convention I've seen a lot of places to move attributes to a new line when they're long as such:
```
<button
  className={`button ${this.props.btnClasses} splash-login-button`}
  ...
>
```
Also note that the last `>` is on a new line (almost like closing braces).

File: app/assets/javascripts/components/login/home_login.jsx [hunk 3]
```diff
@@ -1,19 +1,62 @@
...
+                      onClick={this._handleLogin}>Log In</button>
+            </Modal.Footer>
+          </form>
+        </Modal>
+      </div>
     );
   }
 }
+
+HomeLogin.propTypes = { btnClasses: React.PropTypes.string };
```
Comment (warrenshen) [inline app/assets/javascripts/components/login/home_login.jsx]: It's nice to define default props when a prop is optional. You can do that with `HomeLogin.defaultProps`.

**Pull request merged**